Update dashboard plugin guide - Closes 1858 (#1866)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc

npm run build
----

== Starting the node

[TIP]
====
It is also possible to enable the Dashboard plugin without updating the `plugins.ts` file as suggested in the aforementioned steps.
Once the plugin is installed, you can add the `--enable-dashboard-plugin` flag to the `start` command to run the node.

[tabs]
=====
Sidechain::
+
--
* Start the blockchain client with designated flags on a `sidechain` node.
+
[source,bash]
----
./bin/run start --config=config/custom_config.json --enable-dashboard-plugin
----
--
Mainchain::
+
--
* *Or*, start the blockchain client with designated flags on a `mainchain` node.
For more information, see xref:{url_network_page}[Networks], xref:{url_mainchain_page}[Mainchain] and the xref:{url_coreIndex_page}[Lisk Core] docs.
+
[source,bash]
----
lisk-core start --network testnet --enable-dashboard-plugin
----
--
=====
====

> If you registered the Dashboard via `plugins.ts`, then:

Reviewer: ```suggestion
If you registered the Dashboard via `plugins.ts`, then perform the following:
```

=== Call endpoints

It is possible to invoke endpoints of registered Modules and Plugins via the Dashboard.
Endpoints can be invoked from the *Call endpoint* section. 

In the previous guides, the following endpoints were created:

.*For Module:*
. The endpoint `hello_getHello` is for xref:{url_guides_module_getHello}[Getting the latest Hello for an address].

.*For Plugin:*
. The endpoint `helloInfo_getMessageList` is for xref:{url_guides_plugin_getMessageList}[Getting a list of all Hello messages sent on-chain].

> Let's see, how to invoke each endpoint:

Reviewer: Let's see, how to invoke each endpoint:  imho I would suggest we slightly reword this sentence to the following:
How to invoke each endpoint is covered in the following section:

Author: ```suggestion
To invoke an endpoint through the Dashboard, refer to the following section:
```

.Latest hello message from the given account
image::guides/dashboard/call_action_success.png["Get latest hello message for the given account",1000,align="center"]

==== Invoking `helloInfo_getMessageList`

Select `helloInfo_getMessageList` from the dropdown menu.

.Select the 'helloInfo_getMessageList' endpoint
image::guides/dashboard/call_pluginaction.png["Invoke plugin action",750]

You can directly click on the kbd:[Submit] button to view the results.
The endpoint doesn't require any input for the request to proceed.

.List of all the hello messages sent on-chain
image::guides/dashboard/call_pluginaction_success.png["Plugin action success",1000]

As expected, the Dashboard responds with a list of hello messages that were posted to the Lisk application.

=== Subscribing to events

At the bottom of the Dashboard is the *Recent events* window, which allows the possibility to subscribe to various events by selecting them from a dropdown box.

Select the `hello_newHello` event.

.Subscribe to the 'newHello' event
image::guides/dashboard/subscribe_event.jpg["Subscribe to newHello event", 1000,align="center"]

> Once a new hello transaction is posted, the `newHello` event is also published subsequently and will show up in the *Recent events* window.

Reviewer: ```suggestion
Once a new hello transaction is posted, the `newHello` event is also published, and will subsequently appear in the *Recent events* window.
```

The Dashboard plugin provides various other features as well, most of which are very self-explanatory.
For example, you can create new accounts, see unconfirmed transactions, and see recently generated blocks.

> You can also see general node and network information, etc through the Dashboard plugin.

Reviewer: ```suggestion
It is also possible to view the general node and network information, etc., through the Dashboard plugin.
```
